docs/fp_primer.md

```diff
@@ -446,10 +483,10 @@ literally pages of expanded intervals.
 ```
   x = π/2
 
-  sin(π/2) => [sin(π/2) - 2-11, sin(π/2) + 2-11]
-           => [0 - 2-11, 0 + 2-11]
+  sin(π/2) => [sin(π/2) - 2^-11, sin(π/2) + 2^-11]
```

Comment:
Suggest a note about notation at the top, that in this doc, a^b means `a` to the power of `b`